### ISSUE TITLE
create target should accept directory for --seed arg

### ISSUE THREAD

**Issue by kjk** [fuzzitdev/fuzzit#18]
Currently, the user has 2 to create a `.tar.gz` archive of the corpus in order to submit it with `--seed` arg.

To save this step `--seed` arg should also take a directory name. If it's a directory, it does the creating of `.tar.gz` itself.

Also, should validate that there are no subdirectories (which is probably a mistake) and that there is at least 1 file.

**Comment by yevgenypats** [comment]
Agree. BTW currently the workflow is usually creating the target via dashboard.

**Comment by kjk** [comment]
Understood, but looking from the perspective of submitting instructions to projects about how to integrate their projects with fuzzit, I would like to script as much as possible, so they can re-create the necessary steps easily.

Less they have to use the UI, the better.
